pages/common/perl.md

```diff
@@ -30,3 +30,7 @@
 - Run a multi-line find/replace expression on a file, and save the result in another file:
 
 `perl -p0e 's/{{foo\nbar}}/{{foobar}}/g' {{input_file}} > {{output_file}}`
+
+- Run a PCRE on a piped input, print out first capture group:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a regular expression on stdin, printing out first capture group for each line:
```

Reply:
@sbrl, as per the conversation in #3114, should this remain as stdin?

Reply:
@pxgamer Good shout. I'd say so personally, but I'm not sure what the general consensus is.

Reply:
I'd personally say that "standard input" sounds better here. Otherwise, if using "stdin", since #3114 is already merged I'd say let's enclose it in backticks. We can always change it later if we want, but let's keep it consistent.

Reply:
We could simply say "input from another command". "Standard input" is about as opaque as "stdin" for those who don't know about the concept.

Reply:
Hrm. That's true @waldyrious, but I'd also be wary of using non-standard terminology. It's probably a good thing to get users used to words and phrases like `stdin` and `standard input`, since they are used elsewhere.

In this case though, I think it does sound better to use `input from another command`.